### PR TITLE
Set the test IDs according to the integreatly-test-cases repo

### PR DESCRIPTION
**Pull request by b1zzu**
# Description

According to the test best practices [1] all tests must be linked to the integreatly-test-cases repo [2] by using the same ID

Because some of the tests didn't exist in the integreatly-test-cases repo, I've created a PR [3] also there to add the missing tests

1. https://github.com/integr8ly/integreatly-operator/blob/master/test/BEST_PRACTICES.md#test-case-traceability
2. https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases
3. https://gitlab.cee.redhat.com/integreatly-qe/integreatly-test-cases/merge_requests/101

### CODE REVIEW

File: test/common/tests.go [hunk 1]
```diff
@@ -9,21 +9,21 @@ var (
 
 	AFTER_INSTALL_TESTS = []TestCase{
 		// Test to be execute after RHMI has been installed
```
Comment (davidffrench) [inline test/common/tests.go:11]: Could you put a comment here with a link to the test cases please

Comment (b1zzu) [reply test/common/tests.go:14]: @davidffrench I've added a comment on the file header

**Comment by davidffrench** [comment]
/lgtm
/approve

Thanks @b1zzu

**Pull request merged**